pkg/probes/legacy/legacy.go

```diff
@@ -0,0 +1,162 @@
+package legacy
+
+import (
+	_ "embed"
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	handledErrors "github.com/openshift/osd-network-verifier/pkg/errors"
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+type LegacyProbe struct{}
```

Comment:
nitpick - I personally prefer to not name types starting with the package name, since it's redundant to external callers. e.g. another package calling this would look like `legacy.LegacyProbe`.

Reply:
I hear you; I did feel a twinge of ick while writing that exact reference in the (soon-to-be-PR'd) integration code. Do you have any suggestions of how to avoid?

Reply:
My vote would be to just go with `type Probe struct{}`, and then external callers would do `legacy.Probe` which is still clear to me.

Reply:
Might that conflict with [the Probe interface](https://github.com/openshift/osd-network-verifier/pull/240/files#diff-05c2aa716bb4ebe6725fdbf3d33668763dbec6abc29aad7881e053dc4c07cdf7) though? Probably not literally due to being in different packages (although `legacy` is a child of `probes`), but mentally? Not trying to be obstinate, just want to best understand the suggestion